psi4/src/read_options.cc

```diff
@@ -260,6 +260,42 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_str("PCM_CC_TYPE", "PTE", "PTE");
     }
 
+    /*- PE boolean for polarizable embedding module -*/
+    options.add_bool("PE", false);
+    if (name == "PE" || options.read_globals()) {
+        /*- MODULEDESCRIPTION Performs polarizable embedding model (PE) computations. -*/
+
+        /*- Name of the potential file -*/
+        options.add_str_i("POTFILE", "potfile.pot");
+        /*- Use DIIS acceleration to obtain induced moments -*/
+        options.add_bool("DIIS", true);
+        /*- Threshold for induced moments convergence -*/
+        options.add_int("CONVERGENCE_INDUCED", 8);
```

Comment:
ok to reverse this to `INDUCED_CONVERGENCE` to be more predictable wrt `D_CONVERGENCE`, etc.?

Reply:
Done.